### PR TITLE
Added installation instructions to README

### PR DESCRIPTION
**Pull request by dvisockas**
Once I found the repo, I had to go to the website and click "Getting started" to get installation instructions. Adding them to the README would save some clicks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,36 @@
...
+This will install a package in your configured components folder. You can watch the bower package repository here. As you can see, it's pretty much broken down to things that matter. The raw source. For development as well as production use.
+
+###Installing with Git
+Another way to get the source of angular-translate, is to clone the whole repository from GitHub.
+
+`
+$ git clone git://github.com/PascalPrecht/angular-translate.git
+`
+
+You now have a full clone of the repository including the history and everything else that ever happened during the development of angular-translate. Do with the code what ever the fuck you want.
```
Comment (tspaeth) [inline README.md]: Hm. The last sentence isn't really serious.
And on the other side, the whole git clone won't work for end users as cloning the source repository won't help you in using a distribution and you need (as partly documented below) a compilation step giving you the dist - version.
So to me, this is more something for the Developers rather then for the end users.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,36 @@
...
+$ bower install angular-translate
+`
+
+This will install a package in your configured components folder. You can watch the bower package repository here. As you can see, it's pretty much broken down to things that matter. The raw source. For development as well as production use.
+
+###Installing with Git
+Another way to get the source of angular-translate, is to clone the whole repository from GitHub.
+
+`
+$ git clone git://github.com/PascalPrecht/angular-translate.git
```
Comment (tspaeth) [inline README.md]: Isn't it github.com/angular-translate/angular-translate.git ?

**Comment by dvisockas** [comment]
Frankly, I just copied and formatted this info from http://angular-translate.github.io/docs/#/guide/00_installation

**Comment by tspaeth** [comment]
Oh, ok. Then we should fix this there first :-) - thanks.

**Comment by knalli** [comment]
Well, I don't like to have duplicated content.

I resolve this with adding a link to the installation chapter.

**Comment by dvisockas** [comment]
It is usual that libraries have their installation instructions in their Readme, so people wouldn't have to click and search.

But it's your repo, your rules ;)

**Pull request closed without merging**